src/main/scala/viper/gobra/frontend/info/implementation/typing/ghost/GhostMemberTyping.scala

```diff
@@ -71,4 +78,14 @@ trait GhostMemberTyping extends BaseTyping { this: TypeInfoImpl =>
   }
 
   private def isPurePostcondition(spec: PFunctionSpec): Messages = spec.posts flatMap isPureExpr
+
+  private def nonVariadicArguments(args: Vector[PParameter]): Messages = args.flatMap {
+    p: PParameter => error(p, s"Pure members cannot have variadic arguments, but got $p", p.typ.isInstanceOf[PVariadicType])
+  }
+
+  private def wellDefVariadicArgs(args: Vector[PParameter]): Messages =
+    args.dropRight(1).flatMap {
+      p => error(p, s"Only the last argument can be variadic, got $p instead", p.typ.isInstanceOf[PVariadicType])
+    }
```

Comment:
Why is this not checked in the member typing (for non ghost)?

Reply:
changed in the last commit

Reply:
The reason I did this is that `wellDefActualMember` ends up calling the `wellDefIfPureMethod` and `wellDefIfPureFunction` which are both defined in the `GhostExprTyping.scala`. As such, it just seemed easier to extend the code in that file. I have changed it in the last commit for this particular function